src/sessions/video-session-handler.ts

```diff
@@ -481,34 +483,40 @@ export default class VideoSessionHandler extends BaseSessionHandler {
 
       // if we are unmuting, we need to get a new camera track and add that to the session
     } else {
-      this.log('info', 'Creating new video track', { conversationId: session.conversationId, sessionId: session.id, sessionType: session.sessionType });
+      // Make sure we don't have any tracks before we decide to spin up another.
+      if (session._outboundStream.getVideoTracks().length === 0) {
```

Comment:
This is really the bulk of it. Basically if you spam video mute/unmute it was possible to get multiple unmute requests pending and each would spin up a new video track but each mute request would only mute the first track. 

As this is written, we'll just check to make sure we don't have any video tracks (you shouldn't ever have more than one) before spinning up another.

Reply:
Can we invert this condition so we don't have a massively indented block? Something like:

```
if (session._outboundStream.getVideoTracks().length > 0) {
  this.log('debug', 'Cannot unmute, a video track already exists', { conversationId: session.conversationId, sessionId: session.id, sessionType: session.sessionType });
  return;
}
...
```

the `return` allows us to *not* have an `else` block.

Reply:
Yes, I wanted to do it earlier but had forgotten.